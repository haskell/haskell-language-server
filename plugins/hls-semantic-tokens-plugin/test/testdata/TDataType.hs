module TDataType where

data Foo = Foo Int deriving (Eq)
