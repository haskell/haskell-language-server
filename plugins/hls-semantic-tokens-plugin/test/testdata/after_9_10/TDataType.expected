[1:1-7 HsSyntacticTokenType TKeyword "module"]
[1:8-17 HsSemanticTokenType TModule "TDataType"]
[1:18-23 HsSyntacticTokenType TKeyword "where"]
[3:1-5 HsSyntacticTokenType TKeyword "data"]
[3:6-9 HsSemanticTokenType TTypeConstructor "Foo"]
[3:10-11 HsSyntacticTokenType TKeyword "="]
[3:12-15 HsSemanticTokenType TDataConstructor "Foo"]
[3:16-19 HsSemanticTokenType TTypeConstructor "Int"]
[3:20-28 HsSyntacticTokenType TKeyword "deriving"]
[3:29-30 HsSyntacticTokenType TKeyword "("]
[3:29-30 HsSyntacticTokenType TKeyword "("]
[3:30-32 HsSemanticTokenType TClass "Eq"]
[3:32-33 HsSyntacticTokenType TKeyword ")"]
[3:32-33 HsSyntacticTokenType TKeyword ")"]
